reactive/calico.py

```diff
@@ -247,11 +247,11 @@ def get_mtu(overlay_interface=False):
     if not charm_config('veth-mtu'):
         return None
     if overlay_interface:
-        return charm_config('veth-mtu') if charm_config('ipip') == 'Never' \
-           else (charm_config('veth-mtu') - 50)
-    else:
-        return charm_config('veth-mtu')
-    return None
+        ipip_enabled = charm_config('ipip') != 'Never'
+        vxlan_enabled = charm_config('vxlan') != 'Never'
+        if ipip_enabled or vxlan_enabled:
+            return charm_config('veth-mtu') - 50
```

Comment:
I know this is a carry-over from the initial implementation, but the config option description says that the user should follow the recommendations from the docs, which include the "physical MTU minus 50 if using VXLAN" but the config description doesn't explain that we do that automatically, which might lead users to double-subtract it.  Can you please add a note to the config option.